libraries/stdlib/js/src/org.w3c/org.w3c.dom.events.kt

```diff
@@ -483,3 +483,133 @@ public external interface EventListener {
     fun handleEvent(event: Event): Unit
 }
 
+/**
+ * Exposes the JavaScript [PointerEvent](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent) to Kotlin
+ */
+public external open class PointerEvent(type: String, eventInitDict: PointerEventInit = definedExternally) : MouseEvent {
+    open val pointerId: Int
+    open val width: Int
+    open val height: Int
+    open val pressure: Double
+    open val tangentialPressure: Double
+    open val tiltX: Int
+    open val tiltY: Int
+    open val twist: Int
+    open val pointerType: String
+    open val isPrimary: Boolean
+    fun getCoalescedEvents(): Array<PointerEvent>
+}
```

Comment:
Here are some alternative definitions used in [Chromium](https://github.com/chromium/chromium/blob/master/third_party/blink/renderer/core/events/pointer_event.idl)  and [Gecko](https://github.com/mozilla/gecko-dev/blob/75ec5b5525839e8e8acc11b449ced08a4ce2ef5e/dom/webidl/PointerEvent.webidl) compared to this PR

Attribute | Gecko  |Chromium | This PR
------------ | ------------- | ------------- | -------------
pointerId | long | long | Int
width | long | double | Int
height | long | double | Int
pressure | float | float | Double
tangentialPressure | float | float | Double
tiltX | long | long | Int 
tiltY | long | long | Int
twist | long | long | Int
pointerType | DOMString | DOMString | String (that's fine)
isPrimary | boolean | boolean | Boolean
 getCoalescedEvents() |  sequence<PointerEvent> |  sequence<PointerEvent> | Array<PointerEvent>

Regarding this table, I think it worth to discuss following issues:

1.  Where is *.idl definition. Here's how it works - a corresponding idl-definition is created somewhere in `libraries/stdlib/js/idl`, then you can run gradle idl2k in `libraries/tools/idl2k` and commit generated files - there's some reasoning behind not having autcommited them. We can discuss step 1b more specifically later, however, the first step is still having idl definitions. 

2. I see no reason why we should be more demanding than browser engines, so I suggest to keep float for both `pressure` and `tangentialPressure ` (as of `width` and `height` I think it's fair thing to keep them Int)